.changes/remove-attohttpc.md

```diff
@@ -0,0 +1,6 @@
+---
+'tauri': patch
+'tauri-bundler': patch
+---
+
+Removed `attohttpc` in favor of `reqwest` (core) and `ureq` (tooling).
```

Comment:
I prefer a separate change file for each package in this case.

Reply:
right, makes sense! Still trying to get a feeling for changefiles